src/visidentifier/visprotocol.hpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef VIS_PROTOCOL_HPP_
+#define VIS_PROTOCOL_HPP_
+
+#include <Poco/JSON/Object.h>
+#include <string>
+
+namespace visprotocol {
+constexpr const char* ActionGet            = "get";
+constexpr const char* ActionSubscribe      = "subscribe";
+constexpr const char* ActionSubscription   = "subscription";
```

Comment:
what is a different between ActionSubscribe & ActionSubscription ?

Reply:
Refactored using enum that are easier to understand and GetStriing()